lib/tower_cli/resources/job.py

```diff
@@ -209,6 +209,14 @@ def launch(self, job_template=None, monitor=False, timeout=None,
         if supports_job_template_launch and not use_job_endpoint:
             job_id = job_started.json()['job']
 
+        # If 'ignored_fields' is present in the returning json, display it
+        # in verbose mode.
+        if 'ignored_fields' in job_started.json():
```

Comment:
@AlanCoding Agreed. IMO it is not worthwhile enough for adding extra logic to slow down the process here, My workaround is changing the prompt text: Make it a detail message for user to consult, rather than a warning that explicitly draws user's attention.